### ISSUE TITLE
Feature: Favorite grains (tab)

### ISSUE THREAD

**Issue by xarinatan** [sandstorm-io/sandstorm#3151]
Perhaps semi related to https://github.com/sandstorm-io/sandstorm/issues/2585 
But I think it'd be great to have an extra 'tab' below the Apps and Grains tabs called 'Favorites' or 'Pinned' or something like that.
It would allow you to quickly navigate to grains you use often, without them automatically opening every time you open sandstorm (like the pinned apps suggestion), and without them disappearing in your large list of grains in the Grains tab (like the case is for me right now). 
Alternatively 'favorited' grains could sort before the other non-favorited grains in the Grains tab, that would also work, although is less neat than it having its own tab :)

Thanks for making Sandstorm! I use it daily, and it's a really nice tool to have for people who care about owning their data :heart:

**Comment by ocdtrekkie** [comment]
Yeah, I think some grain organization features like pinning or something would be fantastic to have someday.

**Comment by ocdtrekkie** [comment]
@zenhack This issue does exist, and I put on the UI milestone. :)

**Comment by ocdtrekkie** [comment]
I just was thinking about this today, as my grain I store my "projects to work on" stuff in had fallen pretty far down the grain list, and it made me sad, because I want to see it early and often as a reminder.
